cli/kaos_cli/facades/tests/test_backend_facade.py

```diff
@@ -345,24 +341,19 @@ def test_method_deactivate_context(self, environment, backend, url, token, infra
         # Assert
         self.assertIsNone(active_context)
 
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_method_remove_build_files(self, environment, backend, url, token, infra, kubeconfig, user):
 
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+        test_path = os.path.join(KAOS_STATE_DIR, 'test')
+        os.mkdir(test_path)
 
+        # Act
+        self.facade._remove_build_files(test_path)
 
+        # Assert
+        self.assertFalse(os.path.exists(test_path))
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+        # Rearrange
+        shutil.rmtree(test_path, ignore_errors=True)
```

Comment:
If you have to perform a side-effect, like this filesystem cleanup operation, then this is not a unit-test anymore, this is an integration test. Either reformulate the test and mock the filesystem making this a true unit test, or move this test to the integration tests package, it can still valuable but it shouldn't be here.

Reply:
The filesystem manipulation has now been mocked in both ```test_backend_facade.py```and ```test_state_service.py```